### DISCUSSION TITLE
activemq-artemis-controller-manager pod is unable to start

### DISCUSSION THREAD

**Discussion by vanyagyosheva** [arkmq-org/arkmq-org-broker-operator#648] [General]
Hello, 
I just installed the operaor on aks 1.24.10 using the following script : https://github.com/artemiscloud/activemq-artemis-operator/blob/main/deploy/install_opr.sh, but the pod is unable to start and the log is giving:
I0711 12:32:21.353736       1 request.go:690] Waited for 1.042641828s due to client-side throttling, not priority and fairness, request: GET:https://10.0.0.1:443/apis/installer.devtron.ai/v1?timeout=32s
{"level":"error","ts":"2023-07-11T12:32:30Z","logger":"setup","msg":"failed in detecting openshift","error":"unable to retrieve the complete list of server APIs: packages.operators.coreos.com/v1: the server is currently unable to handle the request","stacktrace":"main.main\n\t/workspace/app/main.go:184\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:250"

Is the operator compatible with aks 1.24.10, everything else seems successfully applied to the cluster

**Comment by gtully** [comment]
that looks like a bug, it should assume kube in that case rather than erroring out with an exit. 
the other option is that there is some permissions problem that prevents that list from returning, I wonder what the api server is saying. Is there a possibility of getting a log from aks?

**Comment by gaohoward** [comment]
Based on this article https://aptakube.com/blog/troubleshooting-api-resource-list you can try to find out if the  packages.operators.coreos.com/v1 is available or not
`kubectl get apiservice`
Can you try that and see what's the output?
